### PR TITLE
The Supplypod Renaissance Part 3: Admins can add shrapnel to supplypods

### PR DESCRIPTION
**Pull request by MrDoomBringer**
## About The Pull Request
Allows adding a pellet_cloud effect to supplypods on landing, available from the config/launch supplypod menu

## Why It's Good For The Game
image

## Changelog
:cl: MrDoomBringer / Ryll-Ryll
admin: Admins can now add shrapnel clouds of any projectile they desire to supplypods launched with the Config/Launch Supplypod menu!
/:cl:

Quick showcase:
image
image

### CODE REVIEW

**Review by stylemistake — DISMISSED** [review]

File: code/modules/cargo/centcom_podlauncher.dm [hunk 1]
```diff
@@ -236,6 +237,23 @@ force_open = FALSE, datum/tgui/master_ui = null, datum/ui_state/state = GLOB.adm
 			temp_pod.desc = descInput
 			temp_pod.adminNamed = TRUE //This variable is checked in the supplypod/setStyle() proc
 			. = TRUE
+		if("effectShrapnel") //Creates a cloud of shrapnel on landing
+			if (temp_pod.effectShrapnel == TRUE) //If already doing custom damage, set back to default (no shrapnel)
```
Comment (bobbahbrown) [inline code/modules/cargo/centcom_podlauncher.dm:241]:
```suggestion
			if (temp_pod.effectShrapnel) //If already doing custom damage, set back to default (no shrapnel)
```

**Review by Cyberboss — DISMISSED** [review]
@MrSupplyPodBringer

**Comment by MrDoomBringer** [comment]
tgstation-server removed the Merge Conflict label now

**Comment by stylemistake** [comment]
There is a runtime which fails the E2E test on travis, seems to be persistent between builds.

**Comment by MrDoomBringer** [comment]
image
scared?

**Pull request merged**